src/Atmos/Model/source.jl

```diff
@@ -38,33 +38,40 @@ struct GeostrophicForcing{FT} <: Source
   u_geostrophic::FT
   v_geostrophic::FT
 end
-function atmos_source!(s::GeostrophicForcing, m::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
-  u = state.ρu / state.ρ
+function atmos_source!(s::GeostrophicForcing, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
+  ρ = state.ρ
+  u = state.ρu/ρ
   u_geo = SVector(s.u_geostrophic, s.v_geostrophic, 0)
-  source.ρu -= state.ρ * s.f_coriolis * (u - u_geo)
+  ẑ = vertical_unit_vector(atmos.orientation, aux)
+  fkvector = s.f_coriolis * ẑ
+  source.ρu -= ρ * fkvector × (u .- u_geo)
 end
 
 """
-  RayleighSponge{FT} <: Source
+    RayleighSponge{FT} <: Source
+
 Rayleigh Damping (Linear Relaxation) for top wall momentum components
 Assumes laterally periodic boundary conditions for LES flows. Momentum components
 are relaxed to reference values (zero velocities) at the top boundary.
 """
 struct RayleighSponge{FT} <: Source
-  "Domain maximum height [m]"
+  "Domain maximum height `[m]`"
   zmax::FT
-  "Vertical extent at with sponge starts [m]"
+  "Vertical extent at with sponge starts `[m]`"
   zsponge::FT
   "Sponge Strength 0 ⩽ c_sponge ⩽ 1"
   c_sponge::FT
+  "Relaxation velocity components"
+  u_relaxation::SVector{3,FT}
 end
-function atmos_source!(s::RayleighSponge, m::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
-  FT = eltype(state)
-  z = altitude(m.orientation, aux)
-  coeff = FT(0)
+function atmos_source!(s::RayleighSponge, atmos::AtmosModel, source::Vars, state::Vars, aux::Vars, t::Real)
+  ρ = state.ρ
+  β = FT(0)
   if z >= s.zsponge
-    coeff_top = s.c_sponge * (sinpi(FT(1/2)*(z - s.zsponge)/(s.zmax-s.zsponge)))^FT(4)
-    coeff = min(coeff_top, 1.0)
+      coeff_top = s.c_sponge * (sinpi(FT(1/2)*(z - s.zsponge)/(s.zmax-s.zsponge)))^FT(4)
```

Comment:
The design docs use an exponent of 2. Which is correct?

cc: @tapios

Reply:
```suggestion
     β_sponge  = s.α_max * (sinpi(FT(1/2)*(z - s.zsponge)/(s.zmax-s.zsponge)))^FT(4)
```

Reply:
> The design docs use an exponent of 2. Which is correct?
> 
> cc: @tapios

Either exponent is justifiable (it's just empirical). Klemp & Lilly (1978) use `2` and base their analysis on that, so let's stick with that (as in the design document):
```suggestion
      β_sponge  = s.α_max * (sinpi(FT(1/2)*(z - s.zsponge)/(s.zmax-s.zsponge)))^FT(2)
```